HiP-UserStore/Utility/EndpointConfig.cs

```diff
@@ -32,5 +32,10 @@ public class EndpointConfig
         /// to the original image files are then returned.
         /// </remarks>
         public string ThumbnailUrlPattern { get; set; }
+
+        /// <summary>
+        /// URL that points to the API of the Data Store
+        /// </summary>
+        public string DataStoreUrlPattern { get; set; }
```

Comment:
1. I think this should be named "DataStoreUrl" instead of "DataStoreUrlPattern" because we expect the full URL here, not a pattern with placeholders or something.
2. The comment should point out whether the URL should end with `/` or not or if it doesn't matter